### PR TITLE
Update notebooks

### PR DESCRIPTION
**Pull request by znicholls**
# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable)
- [x] Example added (either to an existing notebook or as a new notebook, where applicable)
- [x] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <http://link-to-pr.com>`_) Added feature which does something
- (`#XX <http://link-to-pr.com>`_) Fixed bug identified in (`#XX <http://link-to-issue.com>`_)
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: python
 dist: xenial
 sudo: required
 python:
-  - "3.5"
+  # - "3.5"  # incompatible with OpenSCM
```
Comment (rgieseke) [inline .travis.yml]:
```suggestion
```

File: setup.py [hunk 2]
```diff
@@ -80,7 +81,7 @@ def run_tests(self):
         "Development Status :: 4 - Beta",
         "Intended Audience :: Developers",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 3.5",
+        # "Programming Language :: Python :: 3.5",  # OpenSCM incompatible
```
Comment (rgieseke) [inline setup.py]:
```suggestion
```

**Comment by rgieseke** [comment]
@znicholls Pandas should have a `>=0.24.0` requirement in setup.py if removing the hacks requires this version

File: setup.py [hunk 3]
```diff
@@ -47,15 +47,16 @@ def run_tests(self):
...
     "pandas-datapackage-reader",
     "f90nml",
     "pyam-iamc @ git+https://github.com/IAMconsortium/pyam.git@master",
     "PyYAML",
     "expectexception @ git+https://github.com/thedataincubator/expectexception.git@684560f4dfb8cb69eb0693a1605ec0e81fd9b42a",
      "ipywidgets",
      "appmode",
-     "openscm @ git+https://github.com/openclimatedata/openscm.git@add-openscm-dataframe"
+     "openscm @ git+https://github.com/openclimatedata/openscm.git@add-openscm-dataframe",
+     "seaborn"
```
Comment (lewisjared) [inline setup.py]: So any dependencies used in the notebooks get added as explicit dependencies? I mean this already gets installed via seaborn, but I would put the extra dependencies from notebooks as a separate extra requirement target.

Comment (znicholls) [reply setup.py]: @rgieseke and I talked about this a bit, see https://github.com/openclimatedata/pymagicc/pull/198#issuecomment-457640697

I still prefer to have these as explicit dependencies, but happy to change

**Comment by znicholls** [comment]
@rgieseke @lewisjared if you're both happy, I think the series #199, #200, #201, #202 is ready to go then

**Comment by lewisjared** [comment]
I'll leave that one to @rgieseke!

**Comment by znicholls** [comment]
Numerical ie 199, 200, 201, 202

File: setup.py [hunk 4]
```diff
@@ -47,15 +47,16 @@ def run_tests(self):
 cmdclass.update({"test": PyTest})
 
 install_requirements = [
-    "pandas",
+    "pandas>=0.24.0",
     "pandas-datapackage-reader",
     "f90nml",
     "pyam-iamc @ git+https://github.com/IAMconsortium/pyam.git@master",
     "PyYAML",
     "expectexception @ git+https://github.com/thedataincubator/expectexception.git@684560f4dfb8cb69eb0693a1605ec0e81fd9b42a",
```
Comment (rgieseke) [inline setup.py]:
```suggestion
    "expectexception",
```

**Pull request merged**